lib/active_record/connection_adapters/oracle_enhanced_adapter.rb

```diff
@@ -644,6 +644,13 @@ def max_identifier_length
       alias index_name_length max_identifier_length
 
       private
+        def check_version
+          database_version = @connection.database_version.join(".").to_f
+
+          if database_version < 10
+            raise "Your version of Oracle (#{database_version}) is too old. Active Record supports Oracle >= 10g."
```

Comment:
Active Record itself does not "support" Oracle database, Active Record Oracle enhanced adapter does. Then, "Active Record Oracle enhanced adapter support Oracle" may be a good idea.

Reply:
Sure! Oracle enhanced adapter is a 3rd party adapter. I applied your suggestion to this message.